tests/serverpod_test_server/test_integration/test_tools/database_operations_in_endpoint_test.dart

```diff
@@ -82,4 +82,327 @@ void main() {
     },
     runMode: ServerpodRunMode.production,
   );
+
+  withServerpod(
+    'Given TestToolsEndpoint and rollbackDatabase afterEach',
+    (endpoints, session) {
+      group('when calling createSimpleDatasInsideTransactions', () {
```

Comment:
Suggestion: Add to the description that his happens once, otherwise the second test description in this group doesn't really make sense on it's own.

The second test would fail if it is run in isolation right? Should we tag these "scenario" tests in some way to make it clear they are dependent on each other?

Reply:
Maybe I can simply move it into the first test? And yes, I could tag these as scenario. We don't follow that convention elsewhere but maybe we want to start doing that?

Reply:
Sorry I was unclear, I meant moving the `setUpAll` code into the first test.